core/src/navigation/ContextSwitcher.html

```diff
@@ -117,6 +138,7 @@ <h2 class="fd-title fd-title--h5" id="dialog-title-3">
   export let contextSwitcherEnabled = false;
   export let dropDownStates = {};
   export let selectedLabel = null;
+  export let selectedNodePath;
```

Comment:
```suggestion
```
**export** doesn't seem to be needed anymore.  The **getSelectedNode()** alone seems to already resolve the issue 👍 .

I tried removing **export**, and all worked fine for me, so I guess we can remove the export, and leave it as local variable instead below.